sandbox

```diff
@@ -164,17 +165,21 @@ sandbox () {
     curl -s "localhost:8980/health?pretty"
   }
 
+  dc () {
+    docker-compose -f "$SANDBOX_DIR/docker-compose.yml" "$@"
```

Comment:
I believe that docker does some special things to the current working directory. When you use this trick do you need to run it from the same directory each time? For example, if you run `~/sandbox/sandbox up`, can you then change into that directory and run `./sandbox status`?

Reply:
Yes, no problem. Here's running status from a couple directories, neither of them is where I started it.  We used -f thing all the time at the last place I worked and didn't run into any problems.  One thing that gets picked up by docker compose related to the directory is teh default name of the network it instantiates.  But either we're setting that explicitly, or it gets it from wherever -f says the docker-compose.yml file is.


```
➜  ~ $J/sandbox/sandbox status

algod - goal node status
Last committed block: 11152856
Time since last block: 2.8s
Sync Time: 0.0s
Last consensus protocol: https://github.com/algorandfoundation/specs/tree/3a83c4c743f8b17adfd73944b4319c25722a6782
Next consensus protocol: https://github.com/algorandfoundation/specs/tree/3a83c4c743f8b17adfd73944b4319c25722a6782
Round for next consensus protocol: 11152857
Next consensus protocol supported: true
Last Catchpoint:
Genesis ID: testnet-v1.0
Genesis hash: SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=

indexer - health
Indexer disabled for this configuration.
➜  ~ cd Documents
➜  Documents $J/sandbox/sandbox status

algod - goal node status
Last committed block: 11152858
Time since last block: 3.8s
Sync Time: 0.0s
Last consensus protocol: https://github.com/algorandfoundation/specs/tree/3a83c4c743f8b17adfd73944b4319c25722a6782
Next consensus protocol: https://github.com/algorandfoundation/specs/tree/3a83c4c743f8b17adfd73944b4319c25722a6782
Round for next consensus protocol: 11152859
Next consensus protocol supported: true
Last Catchpoint:
Genesis ID: testnet-v1.0
Genesis hash: SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=

indexer - health
Indexer disabled for this configuration.
```